api/common/network.go

```diff
@@ -186,8 +186,14 @@ func interfaceAddressToNetworkConfig(
 	}
 
 	if ipNet := addr.IPNet(); ipNet != nil && ipNet.Mask != nil {
-		config.CIDR = ipNet.String()
+		config.CIDR = network.NetworkCIDRFromIPAndMask(ip, ipNet.Mask)
+	} else {
+		// NOTE(achilleasa): DefaultMask() can return a nil mask; if
+		// that's the case, NetworkCIDRFromIPAndMask will simply return
+		// an empty CIDR.
+		config.CIDR = network.NetworkCIDRFromIPAndMask(ip, ip.DefaultMask())
```

Comment:
I might be trying to be too smart here but I think they will typically be /24 so this should work as expected.